msteams-platform/bots/how-to/conversations/command-bot-in-teams.md

```diff
@@ -0,0 +1,279 @@
+       const machineName = message.matches[1];
+       console.log(machineName);
+       // Render your adaptive card for reply message
+       const cardData = {
+         title: "Your Hello World Bot is Running",
+         body: "Congratulations! Your hello world bot is running. Click the button below to trigger an action.",
+       };
+       const cardJson = AdaptiveCards.declare(helloWorldCard).render(cardData);
+       return MessageFactory.attachment(CardFactory.adaptiveCard(cardJson));
+     }
+   }
+
+   module.exports = {
+     HelloWorldCommandHandler,
+   }
+
+```
+
+### How to build command and response using Adaptive card with dynamic content?
+
+Adaptive Card provides [template language](/adaptive-cards/templating/) to allow users to use dynamic content with the same template. For example, you can use the Adaptive Card to provide a list of items such as to-do list, assigned bugs and other similar lists that can vary according to users. You can perform the following steps to build command and response using Adaptive Card with dynamic content:
+
+1. Add your Adaptive Card template JSON file under `bot/adaptiveCards` folder.
+1. Import the card template in your code file, where the command handler exists such as `myCommandHandler.ts`.
+1. Model your card data.
+1. Use `MessageBuilder.attachAdaptiveCard` in the template with dynamic card data.
+
+You can also add new cards, if required for your application. For more information on how to build different types of Adaptive Cards with a list, or a table of dynamic contents using `ColumnSet`, and `FactSet`, see [sample](https://github.com/OfficeDev/TeamsFx-Samples/tree/ga/adaptive-card-notification).
+
+### Access Microsoft Graph
+
+If you're responding to a command that needs to access Microsoft Graph data of an already signed in Teams user, you can do so by single sign-on (SSO) with their Teams user token. Read more about how Teams Toolkit can help you to add [single sign-on to Teams app](../../../toolkit/add-single-sign-on.md).
+
+### Connect to existing APIs
+
+If you don't have the required SDK, and need to invoke external APIs in your code. The `Teams: Connect to an API` command in Microsoft Visual Studio Code Teams Toolkit extension, or `teamsfx add api-connection` command in TeamsFx CLI can be used to bootstrap code to call target APIs. For more information, see [connect to existing API](../../../toolkit/add-API-connection.md).
+
+## FAQ
+
+<br>
+
+<details>
+
+<summary><b>How to extend my command and response bot to support notifications?</b></summary>
+
+If you need to extend your command and response to support notifications, you can add Adaptive Cards to respond to external events. Follow the steps to customize notifications [Placeholder for customize notification steps in the notification bot in Teams doc].
+
+<br>
+
+</details>
+
+<details>
+
+<summary><b>How to extend my command bot to support Adaptive Card actions?</b></summary>
+
```

Comment:
Can we change `Adaptive Card actions` to `Workflow` so it's consistent with Workflow bot doc.

Reply:
Updated workflow bot in the FAQ.